src/components/Header/Header.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react"
+// import { Link } from "react-router-dom"
+import styles from "../../styles/navbar.scss"
+
+function Header() {
+
+    const [open, setOpen] = useState(false);
+    return (
+        <div>
+            <nav>
+                <div> Logo </div>
+                <ul className={styles["ul-links"]}
+                    style={{ transform: open ? "translateX(0px)" : "" }} >
+                     <li> <a href="/cart">Cart</a> </li>
+                     <li><a href="/contact”"> Contact</a></li >
+               </ul >
+                <div>
+                    <input
+                        className={styles["input"]}
+                        type="text"
+                        placeholder ="search..."
+                    ></input>
```

Comment:
You don't need the second closing tag. Do only `<input className={styles['input'] type="text" ...} />`;